hack/tools/Makefile

```diff
@@ -90,20 +91,24 @@ $(JQ):
 	curl -o $(@) -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 && \
 		chmod a+x $(@)
 
+.PHONY: envtest
+envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
+$(ENVTEST):
+	GOBIN=$(BIN_DIR) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
```

Comment:
nit: maybe pin a version in case of breaking changes? i dont know the state of the tools package here anymore but in the past there were big changes

Reply:
Make sense. Pinned to a version